code/game/machinery/vending.dm

```diff
@@ -27,7 +27,11 @@
 	var/icon_vend
 	/// Icon_state when denying access
 	var/icon_deny
-
+	/// Icons for shared sprites, or overlays that go over a base sprite. If empty, it defaults to the icon_state as the base name for other states.
```

Comment:
`///` is to be used per variable, not a bunch of them. Refer to the [autodoc guide](https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md).